travis-scripts/before_install_osx.sh

```diff
@@ -54,21 +55,19 @@ sudo installer -pkg /Volumes/XQuartz-*/XQuartz.pkg -target /
 popd 
 
 echo "doing an OSX before install step."
-# python may require a little force to install. In October 2014
-# there were issues with easy_install and pip not letting the install
```

Comment:
Side note: You could stop using Homebrew's Python entirely and just use the OS X default one, but the rest of your setup seems to very deliberately use Homebrew's so I left it.